docs/devconsole/README.md

```diff
@@ -26,7 +26,7 @@ $ kubectl create namespace pipelines-{console_username}-github
 $ kubectl create secret -n pipelines-{console_username}-github generic {console_username}-github-token --from-literal=token={user-token}
 ```
 
-6. Application Stages should now be populated with the list of applications.
+6. Environments should now be populated with the list of applications.
 
 ![Application_List](./img/GitOps_Applications.jpg)
```

Comment:
This image has `Application Stages`. Can we replace it with an equivalent `Environments` image?

Reply:
Sorry, must have grabbed the wrong image for that. All good now and deleted the old pictures there as well.